src/clj/fluree/db/query/sparql/translator.cljc

```diff
@@ -366,6 +366,16 @@
   [[_ source]]
   (parse-term source))
 
+(defmethod parse-term :GraphGraphPattern
+  ;; GraphGraphPattern ::= <'GRAPH'> VarOrIri GroupGraphPattern
+  [[_ & [var-or-iri group-graph-pattern]]]
+  [:graph (parse-term var-or-iri) (into [] (parse-term group-graph-pattern))])
```

Comment:
I suspect you don't need the `into` here, since the `:GroupGraphPattern` should return a vector.

Reply:
Without it, I get `({@id ?x, foaf:mbox ?mbox})` instead of `[{@id ?x, foaf:mbox ?mbox}]`

Reply:
Ah, never mind then :+1: